.coveragerc

```diff
@@ -8,11 +8,13 @@ omit =
     # TODO(rhcarvalho): this is used to ignore test files from coverage report.
     # We can make this less generic when we stick with a single test pattern in
     # the repo.
-    test_*.py
-    *_tests.py
+    conftest.py
+    */conftest.py
+    */test_*.py
+    */*_tests.py
 
 [report]
-fail_under = 28
+fail_under = 26
```

Comment:
:+1: this is also what I observe here when the test files are removed from the report